### ISSUE TITLE
re-weighting masking probability

### ISSUE THREAD

**Issue by j-min** [airsplay/lxmert#20]
I found you divide `prob (sampled from 0~1)` with masking ratio again [here](https://github.com/airsplay/lxmert/blob/master/src/pretrain/lxmert_pretrain.py#L85) and [here](https://github.com/airsplay/lxmert/blob/master/src/pretrain/lxmert_pretrain.py#L117).
Could you please explain why you do that?

**Comment by airsplay** [comment]
It follows the original implementation in pytorch transformer [here](https://github.com/huggingface/transformers/blob/b33a385091de604afb566155ec03329b84c96926/examples/lm_finetuning/simple_lm_finetuning.py#L281). 

It re-scales the prob range from [0, 0.15] to [0, 1].

**Comment by j-min** [comment]
Oh I just understood the two-step masking process. Thanks!
